SAPex/Program.cs

```diff
@@ -14,6 +14,19 @@ public class Program
         public static void Main(string[] args)
         {
             CreateHostBuilder(args).Build().Run();
+
+            IConfigurationRoot confRoot = new ConfigurationBuilder().SetBasePath(Directory.GetCurrentDirectory()).Build();
+            string connectionString = confRoot.GetConnectionString("DefaultConnection");
+            var optionsBuilder = new DbContextOptionsBuilder<AppDbContext>();
+            var options = optionsBuilder
+                .UseSqlServer(connectionString)
+                .Options;
+
+            using (AppDbContext db = new AppDbContext(options))
+            {
+                db.TestTable.Add(new TestModel { Name = " first " });
+                db.SaveChanges();
```

Comment:
the same issue with it. write in another place because we should segregate responsibility. in the future we will extract it into other project